templates/src/ConsoleApp/.template.config/template.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Atif Aziz",
+  "classifications": [
+    "Python",
+    "Console"
+  ],
+  "identity": "CSnakes.Templates.Console",
```

Comment:
If you ever plan to release another template for a different language you may want to change this to `CSnakes.Templates.Console.CSharp` so that later you can add `groupIdentity: CSnakes.Templates.Console`. The benefit of this is that you can have a single template that appears as `CSnakes.Tempates.Console` but supports multiple languages and those templates have their own template.config and content. If this will never happen, then no change needed for this field.

Reply:
We've been primarily focusing on C# with source generators, but I guess there's no harm in keeping the doors open here. After all, [we do have a sample in F# too](https://github.com/tonybaloney/CSnakes/tree/1a75be9797b807de0e8372af6ddf43a9c1efb67d/samples/simple/FSharpSample).

Added group identity with be8378edf366f87add0579db2461a0e1d2244db8.